assets/js/components/KeyMetrics/ChipTabGroup/index.js

```diff
@@ -128,14 +159,31 @@ export default function ChipTabGroup( { allMetricItems, savedItemSlugs } ) {
 		[ keyMetricsGroups ]
 	);
 
-	const conversionReportingEventsChange = useSelect( ( select ) => {
+	const newBadgeEvents = useSelect( ( select ) => {
 		if ( ! isGA4Connected ) {
-			return null;
+			return [];
 		}
+		const leadEvents = [ 'contact', 'generate_lead', 'submit_lead_form' ];
```

Comment:
@zutigrm not a blocker for approval at all, but what do you think about moving the lead events to a constant within the file at `assets/js/components/KeyMetrics/constants.js` in case we need to re-use this? I can see this being useful to reference in future.